### ISSUE TITLE
feature request: restrict available localizations/translations

### ISSUE THREAD

**Issue by some-user123** [logto-io/logto#4568]
### What problem did you meet?
Login UI is presented in user's language, even if this language is not supported by the actual application.

### Describe what you'd like Logto to have

Have the ability to restrict the UI to certain languages only that are also supported by my application.

**Comment by charIeszhao** [comment]
You can disable "auto-detect" user language in Sign-in experience configs.

<img width="863" alt="image" src="https://github.com/logto-io/logto/assets/12833674/a85bdfed-3a78-45aa-8e16-77e5819fe617">

**Comment by some-user123** [comment]
Thanks for this advice, but it doesn't completely fit my need.

Assume my application is available in English and Chinese. I want to the login dialogue to be available in English and Chinese (either by autodetect, or by passing the language via query parameter or by manual selection by the user). But I don't want the login dialog ever to show other languages than those two, because:
- I don't want to adapt the texts in all other languages
- I don't want to confuse my users by having the login dialog in a language the app doesn't support

**Comment by Usama4745** [comment]
Hey @charIeszhao is this issue still open to work on?

**Comment by charIeszhao** [comment]
> Hey @charIeszhao is this issue still open to work on?

@Usama4745 Yes, it's still open to work on, and you are always welcome to create a PR to help improve our product.

To @some-user123 This is a valid use case, however, if your customer group only speak English and Chinese, then in most cases they only use these two languages on their browsers, right? Therefore, based on the current "auto detect" logic, they won't see another language, unless they manually set their browser language to a third one (e.g. Japanese), but that's the least thing that is going to happen IMHO. So due to this and also our limited bandwidth, this is going to be a low priority task for the time being.

**Comment by simeng-li** [comment]
Closing this issue due to inactivity.
